feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -161,6 +174,20 @@ export class CustomerListComponent implements OnInit, OnDestroy {
 
       this.asmCustomerListFacade.customerListCustomersSearch(options);
     }
+    this.customerListConfig?.columns?.forEach((item) => {
+      if (
```

Comment:
we maybe make code simpler as below to avoid check both "headerLocalizationKey" and "enableAsmB2bCustomerList" twice?

if (item.headerLocalizationKey === 'asm.customerList.tableHeader.account'  ||   item.headerLocalizationKey === 'hideAccount' )
{
// update column
item.headerLocalizationKey  = this.enableAsmB2bCustomerList ?  'asm.customerList.tableHeader.accoun' : 'hideAccount' ;
}

Reply:
good catch! Addressed as suggestion.